salt/cli/batch_async.py

```diff
@@ -0,0 +1,174 @@
+# -*- coding: utf-8 -*-
+'''
+Execute batch runs
+'''
+
+# Import python libs
+from __future__ import absolute_import, print_function, unicode_literals
```

Comment:
This is fine, the __future__ imports have a special meaning

from https://docs.python.org/2/reference/simple_stmts.html#future-statements:

```A future statement is recognized and treated specially at compile time: Changes to the semantics of core constructs are often implemented by generating different code. It may even be the case that a new feature introduces new incompatible syntax (such as a new reserved word), in which case the compiler may need to parse the module differently. Such decisions cannot be pushed off until runtime.```

Reply:
Thanks for the clarification.